test/integration/crcsuite/crcsuite.go

```diff
@@ -137,9 +137,13 @@ func FeatureContext(s *godog.Suite) {
 
 	s.BeforeScenario(func(pickle *messages.Pickle) {
 		// copy data/config files to test dir
-		err := CopyFilesToTestDir()
-		if err != nil {
-			os.Exit(1)
+		for _, tag := range pickle.GetTags() {
```

Comment:
Is this a 'valid' use of godog tags? (change test behaviour depending on the tag, as opposed as only a way of filtering which tests are run)